### PR TITLE
ui: Redesign - Breadcrumbs styling

### PR DESCRIPTION
**Pull request by kaxcode**
- These styling changes affect many pages throughout the app. 
- No tests were broken by these changes. 
- Chevron-left icon updated to match design mockup.

Design:
Screen Shot 2020-04-22 at 8 35 38 AM

### CODE REVIEW

File: ui-v2/app/styles/base/icons/base-variables.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@ $check-circle-outline-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0
...
 $chevron-down-2-svg: url('data:image/svg+xml;charset=UTF-8,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z" fill="%23000"/></svg>');
 $chevron-down-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6-1.41-1.41z" fill="%23000"/></svg>');
-$chevron-left-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.705 16.885l-4.58-4.59 4.58-4.59-1.41-1.41-6 6 6 6 1.41-1.41z" fill="%23000"/></svg>');
+$chevron-left-svg: url('data:image/svg+xml;charset=UTF-8,<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.705 16.885L11.125 12.295L15.705 7.705L14.295 6.295L8.295 12.295L14.295 18.295L15.705 16.885Z" fill="%23000"/></svg>');
```
Comment (johncowen) [inline ui-v2/app/styles/base/icons/base-variables.scss:30]: This seems to be the uncompressed version of the icon, we shouldn't need to changed this the previous one was slightly more compressed and comes straight from `@hashicorp/structure-icons` - I'd probably say undo this change.

File: ui-v2/app/styles/base/components/breadcrumbs/skin.scss [hunk 2]
```diff
@@ -1,36 +1,19 @@
...
+%crumbs {
+  color: $gray-500;
+  text-decoration: none;
 }
-%breadcrumb::before {
-  line-height: 1;
-  font-size: 0.7em;
-  margin-top: 0.2em;
+%crumbs:hover {
+  color: $cobalt-500;
```
Comment (johncowen) [inline ui-v2/app/styles/base/components/breadcrumbs/skin.scss:6]: I remember one of the designers saying we should never use 'brand colors' for non-brand things, so I think this should be just `$blue-500`.

Comment (kaxcode) [reply ui-v2/app/styles/base/components/breadcrumbs/skin.scss:6]: updated to be $blue-500.

File: ui-v2/app/styles/base/icons/base-variables.scss (hunk 1 above)
Comment (kaxcode) [reply ui-v2/app/styles/base/icons/base-variables.scss]: Put back original SVG
